docs/spl-docs/evaluation-functions/informational-functions.md

```diff
@@ -9,132 +9,252 @@ This function returns `TRUE` if the given value is Boolean.
 
 ### Usage
 
-You can utilize this function with the `eval` and `where` commands and as part of evaluation expressions within other commands.
+`isbool` can be used with `eval` and `where` commands to check if a field's value is a Boolean (`true` or `false`). This is particularly useful in conditional expressions and for filtering records based on Boolean field values.
 
-To determine if a field's value is "true" or "false", use the syntax `<fieldname>=true OR <fieldname>=false` since field values are typically strings or numbers.
+### Example
+
+1. To add a field named `isBoolean` indicating whether the `isActive` field is a Boolean:
 
-### Example 
+```spl
+... | eval isBoolean=if(isbool(isActive), "true", "false")
+```
 
-1. Using the `isbool` function with the `if` function to add a field named `isBoolean` with values "true" or "false" based on whether `isActive` is a Boolean:
+2. To filter records where the `isActive` field is Boolean:
 
 ```spl
-... | eval isBoolean=if(isbool(isActive),"true","false")
+... | where isbool(isActive)
 ```
 
-2. Using the `isbool` function with the `where` command to filter results where the `isActive` field is Boolean:
+### Use-Case Example
+
+**Problem**: In a dataset containing user activity logs, the `isActive` field is supposed to be Boolean but sometimes gets incorrectly inputted as a string or number, leading to inaccurate filtering and analysis.
+
+**Solution**: Use the `isbool` function to identify and filter records where `isActive` is correctly a Boolean. This ensures that only records with valid Boolean values are considered in subsequent analysis or processing steps.
+
+**Implementation**:
 
 ```spl
 ... | where isbool(isActive)
+    | stats count by isActive
 ```
 
+**Explanation**:
+
+- The `where` command filters records to include only those where `isActive` is a Boolean value, using the `isbool` function directly for filtering.
+- The `stats` command then counts the number of records grouped by the `isActive` Boolean status, providing a count of active vs. inactive users based on accurate Boolean data.
+
 ## **isnum(&lt;value&gt;)**
 
 This function returns `TRUE` if the given value is numeric.
 
 ### Usage
 
-You can utilize this function with the `eval` and `where` commands and as part of evaluation expressions within other commands.
+`isnum` can be used with `eval` and `where` commands to check if a field's value is numeric. This is especially useful in conditional expressions and for filtering records based on numeric field values.
```

Comment:
> This is especially useful in conditional expressions and for filtering records based on numeric field values.

I think that's not quite correct, because `isnum` returns a boolean, so you can't use it to filter records based on numeric field values. Unless you mean filtering records that have any (rather than a specific) numeric field value

Reply:
yes, this actually means filtering records that have any numeric field value, I'll update the statement to make it more clear.